opus/application/static_media/js/opus.js

```diff
@@ -151,6 +151,19 @@ var opus = {
 
     oldSurfacegeoTarget: null,
 
+    // Tooltipster settings
+    tooltipsMaxWidth: 680,
+    downloadPaneTooltipsMaxWidth: 350,
+    tooltipsTheme: "op-custom-tooltip-theme",
+    tooltipsDelay: 500,
+    multTooltipsDelay: 1500,
+
+    // the x & y coordinate of the current cursor when moving mouse
+    mouseX: 0,
```

Comment:
is this specifically for tooltips or general purpose?

Reply:
This is tracking the mouse position, and that mouse position will be used to set the proper position for the tooltips. It can be used to other places where we need to track the position of the mouse, but for now, it's only used for setting the the position of the tooltips.

Reply:
perhaps it would be useful to add just a bit more detail per your comment above.